code/datums/components/riding.dm

```diff
@@ -316,12 +316,18 @@
 /datum/component/riding/cyborg/force_dismount(mob/living/M)
 	var/atom/movable/AM = parent
 	AM.unbuckle_mob(M)
+	var/mob/living/silicon/robot/S = AM
+	if(S?.throwcooldown)
```

Comment:
This is probably one of the times that you want it to runtime (No ?), since a component with no parent shouldn't exist.